processing-common/build.gradle.kts

```diff
@@ -3,7 +3,6 @@ plugins {
 }
 
 dependencies {
-    implementation(kotlin("stdlib"))
```

Comment:
Interesting...I think this is something that has been causing me some weird problems as well. Is this all handled by including the Kotlin plugin somewhere? If so, how does the version actually get specified?

Reply:
Yep! I believe the version will match the Gradle plugin:
https://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-rc-released/#stdlib-default

Reply:
OK, that's very good to know.